src/backend/expungeservice/models/charge.py

```diff
@@ -86,3 +87,11 @@ def _felony_class_a(self):
     @staticmethod
     def __strip_non_alphanumeric_chars(statute):
         return re.sub(r'[^a-zA-Z0-9*]', '', statute).upper()
+
+    @staticmethod
+    def __set_section(statute):
+        if len(statute) < 6:
+            return None
+        elif statute[3].isalpha():
+            return statute[0:7]
+        return statute[0:6]
```

Comment:
The method can be renamed to `__get_section` as it returns the section. Alternatively, Line 20 can be moved inside this method.

Reply:
Yes, Line 20 should be inside the method. Thank you. I don't agree that it should be renamed `__get_section` because this not a public method that other classes use to retrieve the section. This method sets the `statute` attribute during initialization, and after that it is not used. However, we could add a getter method that returns the `statute` attribute.

Reply:
After getting some sleep I realize the dilemma of calling it set section as it doesn't actually set the section inside the method

Reply:
perhaps get section is more appropriate, however I feel the naming of methods as get/set should be reserved for accessor methods... Although `__get_section` sounds the most appropriate of the two. What do you think of naming it `__extract_section`? Anyone else have any thoughts on this?

Reply:
I don't think there's anything wrong in naming internal methods get/set. We do have leading underscores to the method names that tells users of this class' instance that they're internal methods. get_section / extract_section: either should be fine. 👍 

Also, Python has the @property decorator to implement accessor methods succinctly.

Reply:
Thanks I created an issue #150 to fix the renaming along with changing the return type on the method. Also I read up on the property decorator and that is very nice. It's exactly what I could use in some places of the code. Thanks for pointing that out.